cfgov/unprocessed/css/molecules/inset.less

```diff
@@ -46,6 +68,83 @@
         margin-right: unit( @grid_gutter-width / @base-font-size-px, em );
     });
 
+    &__image {
+        float: right;
+
+        div.m-inset_image {
+            background-size: contain;
+            background-repeat: no-repeat;
+        }
+
+        .m-inset_text {
+            margin-top: unit( 5px / @base-font-size-px, em )
+        }
+
+        .respond-to-max(@bp-xs-max, {
+            margin-left: 0;
+            margin-right: 0;
+
+            .m-inset_image {
+                .u-flexible-container_inner();
+            }
+
+            img.m-inset_image {
+                height: auto;
+                width: auto;
+            }
+
+            div.m-inset_image {
+
+                // Important is necessary to override the inline height value;
+                height: 100% !important;
+
+                &__cover {
+                    background-size: cover;
+                }
+            }
+
+            .m-inset_image-container {
+                overflow: hidden;
+                .u-flexible-container();
+            }
+        });
+
+        .respond-to-min(@bp-sm-min, {
+            &__150 {
+                width: 150px;
+            }
+
+            &__270 {
```

Comment:
Removed in https://github.com/cfpb/cfgov-refresh/pull/2678/commits/98cde738538340d61bc985dcc16f0bdb95da36c9.